trailmet/datasets/classification/cifar.py

```diff
@@ -190,4 +190,4 @@ def __init__(self, name=None, root=None,
 
 
```

Comment:
Avoid adding these line breaks

Reply:
i dont know what is causing these,  these appeared just by opening and closing the files without changing anything in them.